nft/nfts.yaml

```diff
@@ -277,6 +277,8 @@ paths:
           description: String - Contract address for the NFT collection
         - $ref: nfts_body.yaml#/withTokenBalances
           description: Boolean - [boolean]  false by default. If set to true the query will include the token balances per token id for each owner.
+        - $ref: nfts_body.yaml#/block
+          description: String - The block number (in hex or decimal) at which to fetch collection ownership for.
```

Comment:
I'd recommend making this a bit more clear: "String - The point in time or block number (in hex or decimal) to fetch collection ownership information for."